website/docs/guides/version-6-upgrade.html.markdown

```diff
@@ -19,6 +19,7 @@ Upgrade topics:
 - [Dropping Support For Amazon SimpleDB](#dropping-support-for-amazon-simpledb)
 - [Dropping Support For Amazon Worklink](#dropping-support-for-amazon-worklink)
 - [AWS OpsWorks Stacks End of Life](#aws-opsworks-stacks-end-of-life)
+- [resource/aws_instance](#resourceaws_instance)
```

Comment:
I would alphabetize this. Or do we do them in order or RUM usage? This is one of the most important so I can see putting it first.